src/App.tsx

```diff
@@ -17,18 +58,34 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                selectedOption={selectedOption}
+                searchQuery={searchQuery}
+                onChangeSelect={setSelectedOption}
+                onChangeSearch={setSearchQuery}
+              />
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {
+                todos.length === 0
+                  ? <Loader />
```

Comment:
what if server will return empty array?

create and use isLoading state for Loader conditional render